docs/DeveloperGuide.md

```diff
@@ -296,6 +296,151 @@ The following sequence diagram shows how the flow of the example execution:
       * Many commands for user to remember, although they have similar ideas (e.g. `addPerson`, `addLesson` both involving adding a new object, which the user could have used a generic `add` method.)
       * Commands have to manually call the Ui to specify which Ui panels to show.
 
+### EditPersonCommand and Parser
+
+#### Background
+To boost usability and support more functionalities, many commands in TutorMate support multiple forms of valid user input.
+
+For example, when editing a person, the "INDEX" parameter can be conditionally omitted when editing the currently shown entry, the "edit" command word is overloaded for both "editLesson" and "editPerson" command, and for each of these two commands, many optional flags are available.
```

Comment:
Possibly reformat the parameter and command word to the code segments.

Reply:
Thank you for pointing that out. Fixed in the latest commit.